go-controller/pkg/ovn/pods.go

```diff
@@ -458,7 +458,7 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
 	}
 
 	// if we have any external or pod Gateways, add routes
-	gateways := make([]*gatewayInfo, 0, len(routingExternalGWs.gws) + len(routingPodGWs))
+	gateways := make([]*gatewayInfo, 0, len(routingExternalGWs.gws)+len(routingPodGWs))
```

Comment:
@trozet should this be part of the first commit in this PR?

Reply:
yeah gofmt thing, will fix